wikichange/src/content/graph.js

```diff
@@ -1,86 +1,86 @@
-import Chart from "chart.js/auto";
```

Comment:
Same question for this file: why include it if no change?

Reply:
yea this is weird. a quick google search told me that there may be some meta-data changes. if nothing is actually changed from this file, can u revert this (and others) back to what they were?

Reply:
I Ran `git diff main:wikichange/src/content/graph.js wikichange/src/content/graph.js` on this file against the main branch. it turns out that you added a line separator at the end of every line. can you remove them or revert them?

<img width="725" alt="image" src="https://user-images.githubusercontent.com/55929961/213620221-cc3c5139-5ea0-43ec-be50-602a0334e05b.png">

Reply:
I reverse a big change I made so I think my IDE automatically format the old file like this. I am looking into how to reverse the change correctly.

Reply:
maybe try this? https://dev.to/lofiandcode/git-and-github-how-to-revert-a-single-file-dha